Dockerfile

```diff
@@ -9,32 +9,18 @@ COPY ["go.mod", "go.sum", "./"]
 RUN go mod download
 
 USER root
-# default NodeJS 14 is not enough for openapi-generator-cli, switch to Node JS currently supported
-RUN yum remove -y nodejs npm
-RUN yum module -y reset nodejs
-RUN yum module -y enable nodejs:18
-# install npm and java for openapi-generator-cli
-RUN yum install -y nodejs npm java-11
```

Comment:
these in the past proved "painful" to be done in docker merely for producing the end image, and I'm so glad to see them removed _from here_ 🥹

if we wanted easier DX we can always have these in a [DevContainer](https://containers.dev/) instead in the future, wdyt ?